### PR TITLE
chore: remove @ts-ignore in src/api/web/endpoint/package.ts

### PR DESCRIPTION
**Pull request by iketiunn**
Remove @ts-ignore in this file.

Also noticed some typo here, should leave these to another commit:
- PackcageExt -> PackageExt
- packgesCopy -> packagesCopy

### CODE REVIEW

**Review by juanpicado — APPROVED** [review]
All good by far!

Since this issue is present also on branch `5.x` (future verdaccio) you might want also to fix it there. Unfortunately, we are not rebasing from master anymore.

**Comment by iketiunn** [comment]
Np, will work on it later.

**Comment by juanpicado** [comment]
I will take care of this soon, I am bit busy 🙏

File: src/api/web/endpoint/package.ts [hunk 1]
```diff
@@ -119,8 +118,7 @@ function addPackageWebApi(route: Router, storage: IStorageHandler, auth: IAuth,
           const {v} = req.query;
           let sideBarInfo: any = _.clone(info);
           sideBarInfo.versions = convertDistRemoteToLocalTarballUrls(info, req, config.url_prefix).versions;
-          if (isVersionValid(info, v)) {
-            // @ts-ignore
+          if (typeof v === 'string' && isVersionValid(info, v)) {
```
Comment (juanpicado) [inline src/api/web/endpoint/package.ts:121]: I just noticed that `isVersionValid` already check for the type and seems redundant, you might replace the `undefined` check by the `string` and have the whole check within the method.

**Comment by iketiunn** [comment]
Since `5.x` is the way to go, probably I should close this one and polish more on the `5.x` branch.

**Comment by juanpicado** [comment]
Still valid if you want to reopen, `4.x` will be live for quite a while. But as you prefer :)

**Pull request closed without merging**